thrall/test/lib/elasticsearch/ElasticSearchTest.scala

```diff
@@ -1,17 +1,17 @@
 package lib.elasticsearch
 
-import java.util.UUID
-
+import com.gu.mediaservice.lib.elasticsearch.MappingTest
```

Comment:
Are these real changes or refactoring artifacts?  Looks like the whole file could be removed from this PR?